crates/iota-sdk/src/iota_client_config.rs

```diff
@@ -79,17 +79,18 @@ impl IotaClientConfig {
     }
 
     /// Get the first [`IotaEnv`] or one by its alias.
-    pub fn get_env(&self, alias: &Option<String>) -> Option<&IotaEnv> {
-        if let Some(alias) = alias {
-            self.envs.iter().find(|env| &env.alias == alias)
-        } else {
-            self.envs.first()
-        }
+    pub fn get_env(&self, alias: &str) -> Option<&IotaEnv> {
```

Comment:
I don't really know how to feel about this change tbh

Reply:
How so?